docs/api/conf.py

```diff
@@ -67,9 +67,9 @@
 #html_theme = 'alabaster'
 
 html_theme = 'inrupt'
-html_theme_path = ['./build/docs-assets/themes']
+html_theme_path = ['./docs-assets/themes']
 
-html_copy_source = False
+html_copy_source = True
```

Comment:
Q: Curious, but why are we setting this option to True?

Reply:
Debugging, was trying to figure out the links issue